llvm/lib/CodeGen/SelectionDAG/LegalizeTypes.cpp

```diff
@@ -910,6 +910,38 @@ SDValue DAGTypeLegalizer::CreateStackStoreLoad(SDValue Op,
   return DAG.getLoad(DestVT, dl, Store, StackPtr, MachinePointerInfo(), Align);
 }
 
+static SDValue MaybeBitcast(SelectionDAG &DAG, SDLoc DL, EVT VT,
```

Comment:
Remove this and just use DAG.getBitcast()

Reply:
Done